### ISSUE TITLE
Flutter Web UnimplementedError: getServiceStatusStream() Error

### ISSUE THREAD

**Issue by matteoberla** [Michael-M-aher/location_picker_flutter_map#38]
Hi, i'm trying to use this package but when i run the widget an:
"UnimplementedError: getServiceStatusStream() has not been implemented." is thrown.

Similar issue was already found as mentioned here: https://github.com/tlserver/flutter_map_location_marker/issues/101

**Comment by matteoberla** [comment]
after some more research the offending part is the following:

```
Widget _buildCurrentLocation() {
    return CurrentLocationLayer(
      style: const LocationMarkerStyle(
        markerDirection: MarkerDirection.heading,
        headingSectorRadius: 60,
        markerSize: Size(18, 18),
      ),
    );
  }
```

as it tryes to subscribe to the current position stream but failing in web platform.
By setting showCurrentLocationPointer = false the error is not showed anymore but this workaround should be documented.

**Comment by tlserver** [comment]
This issue is fixed in flutter_map_location_marker v8.0.5.

**Comment by Michael-M-aher** [comment]
Fixed ✅
